lib/public_api_test.ts

```diff
@@ -17,6 +17,27 @@ import { clearStorageBeforeAndAfter } from "../testing/storage";
 describe("FledgeShim", () => {
   clearStorageBeforeAndAfter();
 
+  describe("constructor", () => {
+    it("should create an invisible iframe", () => {
+      create();
+      const iframe = document.querySelector("iframe");
+      assertToBeTruthy(iframe);
+      expect(iframe.src).toBe(new URL("/frame.html", document.baseURI).href);
+      expect(getComputedStyle(iframe).display).toBe("none");
+    });
+
+    it("should create a sandboxed iframe", () => {
+      create();
+      const iframe = document.querySelector("iframe");
+      assertToBeTruthy(iframe);
+      expect(iframe.src).toBe(new URL("/frame.html", document.baseURI).href);
+      assertToBeTruthy(iframe.contentDocument);
+      const script = iframe.contentDocument.createElement("script");
+      script.textContent = "top.location.pathname = '/favicon.ico';";
+      iframe.contentDocument.head.appendChild(script);
```

Comment:
I don't understand how this verifies the iframe is sandboxed?

Reply:
Added an explanatory comment.